frontend/src/components/Tag/tag.module.css

```diff
@@ -1,7 +1,10 @@
 .reducedTag {
   height: 11px;
   width: 11px;
-  float: left;
+  display: inline-block;
+  margin-left: 6px;
+  margin-right: 5px;
```

Comment:
Is there special considerations on these values?
I see the width is already 11px.

Reply:
yes they look good and work well (responsively)